src/LCT.PackageIdentifier/NpmProcessor.cs

```diff
@@ -247,7 +247,7 @@ private static void CheckAndAddToBundleComponents(List<BundledComponents> bundle
 
                 components.Description = folderPath;
                 components.Version = Convert.ToString(properties[Version]);
-                components.Author = prop?.Value[Requires]?.ToString();
```

Comment:
why this null check is removed

Reply:
Because you see above it's already been used as prop.name. If the property value was null it wouldn't have reached this step.
All this changes have made with respect to sonar qube code quality issues.